internal/rbd/nodeserver.go

```diff
@@ -869,3 +869,58 @@ func (ns *NodeServer) xfsSupportsReflink() bool {
 	xfsHasReflink = xfsReflinkNoSupport
 	return false
 }
+
+// NodeGetVolumeStats returns volume stats.
+func (ns *NodeServer) NodeGetVolumeStats(ctx context.Context, req *csi.NodeGetVolumeStatsRequest) (*csi.NodeGetVolumeStatsResponse, error) {
+	var err error
+	targetPath := req.GetVolumePath()
+	if targetPath == "" {
+		err = fmt.Errorf("targetpath %v is empty", targetPath)
+		return nil, status.Error(codes.InvalidArgument, err.Error())
+	}
+
+	stat, err := os.Stat(targetPath)
+	if err != nil {
+		return nil, status.Errorf(codes.InvalidArgument, "failed to get stat for targetpath %q: %v", targetPath, err)
```

Comment:
`failed to get stats`..? (s/stat/stats)

Reply:
not really, it is `struct stat` from the `stat()` system call